bot/cogs/trivia.py

```diff
@@ -0,0 +1,307 @@
+"""Commands for the trivia module."""
+import json
+import os
+
+from discord import Colour, Embed, Reaction
+from discord.abc import User
+from discord.ext import commands
+from discord.ext.commands.bot import Bot
+from discord.ext.commands.context import Context
+from random import shuffle
+from html import unescape
+from asyncio import TimeoutError
+
+EMBED_COLOR = Colour.blue()
+
+
+class Trivia(commands.Cog):
+    """commands for the trivia game."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.TOKEN = "null"
+        if os.path.exists("bot/scores.json"):
+            with open("bot/scores.json") as file:
+                self.scores = json.load(file)
+                print('Loaded saved scores')
+        else:
+            self.scores = {}
+            with open("bot/scores.json", "w+") as file:
+                json.dump(self.scores, file, indent=4)
+            print('Creating Blank Scores')
+
+    @commands.command(
+        brief="Takes in an optional argument for category and returns a question.",
+        description="Do .trivia categories to see avaliable choices."
```

Comment:
This would be better as the kwarg `usage`, the description should be a more detailed version of the brief, it should usually describe what the command does.

I would recommend a `usage` kwarg that shows general usage, the description could still describe how to list categories.